hs_core/views/__init__.py

```diff
@@ -2132,6 +2132,7 @@ def metadata_review(request, shortkey, action, uidb36=None, token=None, **kwargs
             f"This resource does not have a pending metadata review for you to { action }.",
         )
     else:
+        res.raccess.alter_review_pending_flags(initiating_review=False)
```

Comment:
@sblack-usu this is the real "hotfix" for 2.12.2 in this PR. The other modifications were just additions to a management command

Reply:
This is the issue that 2.12.2 hotfix was minted for:
https://github.com/hydroshare/hydroshare/issues/5312